### PR TITLE
Fixes GH-542-Moved : Request for Documentation: Add to interactive-messages documentation

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
This PR adds the a very important documentation line, which guides user to how to get the selected value of option of post type Message Menus

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-developer-documentation/issues/542

### CODE REVIEW

File: source/developer/interactive-messages.rst [hunk 1]
```diff
@@ -205,7 +205,7 @@ URL
   The actions are backed by an integration that handles HTTP POST requests when users click the message button. The URL parameter determines where this action is sent. The request contains an ``application/json`` JSON string. As of 5.14, relative URLs are accepted, simplifying the workflow when a plugin handles the action.
 
 Context
-  The requests sent to the specified URL contain the user ID, post ID, channel ID, team ID, and any context that was provided in the action definition. The post ID can be used to, for example, delete or edit the post after clicking on a message button.
+  The requests sent to the specified URL contain the user ID, post ID, channel ID, team ID, and any context that was provided in the action definition. If the post was of type "Message Menus", then context also contains selected_option field with the user selected option value. The post ID can be used to, for example, delete or edit the post after clicking on a message button.
```
Comment (justinegeffen) [inline source/developer/interactive-messages.rst]:
```suggestion
  The requests sent to the specified URL contain the user ID, post ID, channel ID, team ID, and any context that was provided in the action definition. If the post was of type ``Message Menus``, then context also contains the ``selected_option`` field with the user-selected option value. The post ID can be used to, for example, delete or edit the post after clicking on a message button.
```

Comment (justinegeffen) [reply source/developer/interactive-messages.rst]: Minor format changes. Let me know if the proposed changes work. :)

**Comment by M-ZubairAhmed** [comment]
Opps sorry i re requested review @justinegeffen

**Comment by justinegeffen** [comment]
No problem at all, thank you! It looks good to me. :)

**Pull request merged**